content/ruby/concepts/argf/argf.md

```diff
@@ -0,0 +1,130 @@
+---
+Title: 'ARGF and ARGV'
+Description: 'ARGF is a module in Ruby used for reading files passed as command line arguments or in standard input (STDIN) in a terminal. A Ruby script using ARGF may be thought of as implementing a subset of the functions of the Unix command `cat`.'
+Subjects:
+  - 'Web Development'
+Tags:
+  - 'ARGF'
```

Comment:
None of these tags are in the tags.md file. You need to select tags from that list. While you can add new tags in this PR to tags.md, "ARGF" and "ARGV" are too niche to be good tags for an entry. There is an "Input" tag and a "Files" tag that would be appropriate.